### PR TITLE
Add content to Mascot commission page

### PR DESCRIPTION
**Pull request by patocallaghan**
#### Left (existing page) vs Right (new page)

image

This is a work-in-progress for https://github.com/ember-learn/ember-website/issues/21

It's not there yet but I wanted to push it up to get feedback

/cc @mansona  

There's a lot of improvements and optimizations that can be done but I wasn't sure what was "good enough" considering the site is going to be overhauled soon for the redesign.

### CODE REVIEW

File: app/styles/global/_form.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
+.tomster-form {
```
Comment (patocallaghan) [inline app/styles/global/_form.scss]: I've literally copy-and-pasted the existing styles from the current site on dumped them in here. There's been no attempt to try and optimize these styles.

File: app/styles/pages/_mascots.scss [hunk 2]
```diff
@@ -75,7 +75,7 @@
     min-width: inherit;
 }
 
-.mascots-faq {
+.mascots {
```
Comment (patocallaghan) [inline app/styles/pages/_mascots.scss]: To be able to reuse the content/sidebar combination already in use on the FAQ page I've renamed the class to be just `.mascots`. While it works the resulting sidebar is slightly larger than what is on the existing site.

File: app/templates/mascots/commission.hbs [hunk 3]
```diff
@@ -1 +1,130 @@
...
+<section>
+
+  <h1>Mascot Commission Requests</h1>
+
+  <div class="left-col">
+    <p>Thanks so much for your interest in commissioning a Tomster or Zoey for your User Group (UG) or Company (Corp)! As per our {{link-to "FAQ" "mascots.faq"}}, assuming an approved use case, we'd love to work with you!</p>
+    <p>Most of the Tomsters and Zoeys we design fit into a somewhat standard mold. You can <a href="/pdfs/Mascot-Pricing.pdf">download the pricing sheet</a> to learn about the standard options.</p>
+    <p>Filling out this form is the first step to commissioning your Custom Ember Mascot. We'll get back to you as soon as we can with our thoughts and next steps.</p>
+
+    <form id="tomster-commission-form" name="tomster-commission-form" class="tomster-form wufoo topLabel page" accept-charset="UTF-8" autocomplete="off" enctype="multipart/form-data" method="post" novalidate action="https://tilde.wufoo.com/forms/q14mw9qi196f6oy/#public">
```
Comment (patocallaghan) [inline app/templates/mascots/commission.hbs]: I've just copied-and-pasted the existing form markup from the current site. I've made no attempt at rewriting or optimizing the HTML.

File: app/styles/global/_form.scss (hunk 1 above)
Comment (MelSumner) [reply app/styles/global/_form.scss]: that's fine- our goal is to convert then iterate to make better

**Comment by patocallaghan** [comment]
Thanks @MelSumner. I've updated to fix the linting errors, just Percy failing now. I've yet to do the commission-sent page listed in https://github.com/ember-learn/ember-website/issues/21 but I'll do that in a separate PR.

**Comment by MelSumner** [comment]
Thank you for doing this, @patocallaghan :+1:

**Pull request merged**